src/trackers/UNIT3D/skipthecommercials.py

```
@@ -56,9 +57,17 @@ async def get_type_id(
         return {"type_id": type_id}
 
     async def get_additional_checks(self, meta: Meta) -> bool:
-        if str(meta.category) != "TV":
+        category = str(meta.category).upper()
+        if isinstance(meta.combined_genres, list):
+            combined_genres = meta.combined_genres
+        else:
+            combined_genres = [genre.strip() for genre in str(meta.combined_genres).split(",") if genre.strip()]
```

Ruff (0.16.3)

[warning] Use ternary operator `combined_genres = meta.combined_genres if isinstance(meta.combined_genres, list) else [genre.strip() for genre in str(meta.combined_genres).split(",") if genre.strip()]` instead of `if`-`else`-block

Replace `if`-`else`-block with `combined_genres = meta.combined_genres if isinstance(meta.combined_genres, list) else [genre.strip() for genre in str(meta.combined_genres).split(",") if genre.strip()]`

(SIM108)